lib/Debian/Debhelper/Dh_Epics.pm

```diff
@@ -18,7 +18,16 @@ sub setepicsenv {
         $ENV{EPICS_BASE} = "/usr/lib/epics";
     }
     if (not exists $ENV{EPICS_HOST_ARCH}) {
-        $ENV{EPICS_HOST_ARCH} = `$ENV{EPICS_BASE}/startup/EpicsHostArch.pl`;
+        my $EHA_PATH;
+        if (-r "$ENV{EPICS_BASE}/lib/perl/EpicsHostArch.pl") {
+            $EHA_PATH = "$ENV{EPICS_BASE}/lib/perl/EpicsHostArch.pl"
```

Comment:
```suggestion
            $EHA_PATH = "$ENV{EPICS_BASE}/lib/perl/EpicsHostArch.pl";
```

Reply:
Good catch. I updated my PR to include the missing semicolon.